MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController+Design.swift

```diff
@@ -0,0 +1,114 @@
+        contentView.addSubview(castView)
+        
+        let recommendations: (() -> [MovieViewModel]) = { [weak self] in
+            guard let self = self else { return [] }
+            
+            return self.presenter.getRecommendations(for: self.movieId)
+        }
+        recommendationsView = RecommendationsView(getRecommendations: recommendations)
+        contentView.addSubview(recommendationsView)
+    }
+    
+    func styleViews() {
+        view.backgroundColor = .white
+        
+        scrollView.contentSize = CGSize(width: UIScreen.main.bounds.width, height: 1000)
+        scrollView.showsVerticalScrollIndicator = false
+        scrollView.isUserInteractionEnabled = true
+        scrollView.isExclusiveTouch = true
+                        
+        let backButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
+        navigationController?.navigationBar.topItem?.backBarButtonItem = backButtonItem
+        navigationItem.titleView = UIImageView(image: UIImage(with: .navigationBarTitleImage))
+        
+        overviewTitle.text = "Overview"
+        overviewTitle.font = .heading1
+        overviewTitle.textColor = .primaryBlue
+        
+        overview.text = presenter.getOverview(for: movieId)
+        overview.font = .regularMedium
+        overview.textColor = .black
+        overview.numberOfLines = 0
+        overview.lineBreakMode = .byWordWrapping
+        overview.setLineSpacing(lineSpacing: 0, lineHeightMultiple: 1.4)
+    }
+    
+    func defineLayoutForViews() {
+        scrollView.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+        }
+        
+        contentView.snp.makeConstraints {
+            $0.width.equalTo(UIScreen.main.bounds.width)
+            $0.top.equalToSuperview()
+            $0.height.equalTo(1000)
+        }
+        
+        titleView.snp.makeConstraints {
+            $0.top.equalToSuperview()
+            $0.height.equalTo(300)
+            $0.leading.trailing.equalToSuperview()
+        }
+        
+        overviewTitle.snp.makeConstraints {
+            $0.top.equalTo(titleView.snp.bottom).offset(5 * offset)
+            $0.leading.equalToSuperview().offset(4 * offset)
+        }
+        
+        overview.snp.makeConstraints {
+            $0.top.equalTo(overviewTitle.snp.bottom).offset(3 * offset)
+            $0.leading.equalToSuperview().offset(4 * offset)
+            $0.trailing.equalToSuperview().inset(4 * offset)
+        }
+        
+        castView.snp.makeConstraints {
+            $0.top.equalTo(overview.snp.bottom).offset(8 * offset)
+            $0.leading.trailing.equalToSuperview()
+            $0.height.equalTo(CastView.height)
+        }
+        
+        recommendationsView.snp.makeConstraints {
+            $0.top.equalTo(castView.snp.bottom).offset(8 * offset)
+            $0.leading.trailing.equalToSuperview()
+            $0.height.equalTo(RecommendationsView.height)
```

Comment:
If you constraint this to the bottom, you don't need to set the scroll view content height.

Reply:
[Here](https://github.com/lihter/Movie-App/pull/28/commits/3a1dd8418b7535a8fac5faf7eea2183f097a6eef#diff-0e250b879344a24765e175866cb828b87d12e258a723eb41dcf4344c1dcd19fdR108)